policy-controller/k8s/status/src/index.rs

```diff
@@ -19,6 +24,7 @@ use tokio::{
 
 pub(crate) const POLICY_API_GROUP: &str = "policy.linkerd.io";
 const POLICY_API_VERSION: &str = "policy.linkerd.io/v1alpha1";
+const PATCH_TIMEOUT: Duration = Duration::from_secs(5);
```

Comment:
While we're here, why don't we expose this through the main's Args so we have an escape hatch if we ever regret this default.